### PR TITLE
Make your first deploy

### PR DESCRIPTION
**Pull request by mdobosz3**
Acceptance Criteria

- You run seeds on production.

- The app is online and accessible in any browser

- There is information in the Github repo's README on how to access the production and sign in.

### CODE REVIEW

**Review by ciejj — APPROVED** [review]
Looks good - good job! 👏

File: README.md [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
+### How to access to the website:
 
-### How to start the app
-```
-rails server
-```
+https://employee-recognition-platfrom.herokuapp.com
+
+### If you want to login please use this account: 
+- login: jedynka@gmail.com
```
Comment (oskarlakner) [inline README.md:8]: I'd suggest using eng names

File: README.md [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
 
-### How to start the app
-```
-rails server
-```
+https://employee-recognition-platfrom.herokuapp.com
+
+### If you want to login please use this account: 
+- login: jedynka@gmail.com 
+- password: 111111
```
Comment (oskarlakner) [inline README.md:9]: IMO you're forcing someone to count `1`s. WDYT about setting it to `password`?

File: README.md (hunk 1 above)
Comment (mdobosz3) [reply README.md]: I changed accounts to english names

File: README.md (hunk 2 above)
Comment (mdobosz3) [reply README.md]: It will definitely make logging in easier for the person who enters for the first time. I changed the data according to your suggestion

**Pull request merged**